test/unittest/tests/wrappers/config_parser.cpp

```diff
@@ -0,0 +1,37 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+#include "config_parser.h"
+#include <string>
+
+config_parser::config_parser(std::map<std::string, std::string> map) : _config_map(map)
```

Comment:
To (a) avoid an unnecessary copy of map, and to make the initialisation of _cfg cleaner we could write the constructor like this:

```
config_parser::config_parser(std::map<std::string, std::string> map) :
  _config_map(std::move(map)),
  _cfg{nullptr, nullptr, nullptr}
{
}
```